ansible/roles/openstack-ansible-os_triliovault_wlm/tasks/install-centos.yml

```diff
@@ -90,42 +90,95 @@
     name: workloadmgr
     state: latest
 
-- name: Install python3-s3-fuse-plugin when backup target is S3
+- name: Install python3-s3fuse-plugin
   package:
-    name: python3-s3-fuse-plugin
+    name: python3-s3fuse-plugin
     state: latest
   when: 
     - NFS == False
-    - PYTHON_VERSION=="python3"
 
 - import_tasks: create_conf_dirs.yml
 
 - name: create wlm_logging.conf file
   template:
      src: wlm_logging.conf.j2
      dest: "{{ wlm_log_cfg }}"
+  tags:
+    - update_cfg
+
+- name: create object_store_logging.conf file
+  template:
+     src: object_store_logging.conf.j2
+     dest: "{{ OBJECT_STORE_LOG_CFG }}"
+  when: 
+    - NFS == False
+  tags:
+    - update_cfg
 
 - import_tasks: get_cloud_details.yml
 
+
 - name: copy workloadmgr conf
   template:
      src: triliovault-wlm.conf.j2
      dest: "{{ wlm_cfg }}"
+  tags:
+    - update_cfg
 
 - name: Copy workloadmgr api-paste-ini 
   template:
      src: api-paste.ini.j2
      dest: "{{ wlm_api_paste }}"
+  tags:
+    - update_cfg
 
 - name: Copy triliovault object store conf
   template:
      src: triliovault-object-store.conf.j2
      dest: "{{ triliovault_obj_sotre_cfg }}"
   when: NFS == False
+  tags:
+    - update_cfg
+
+- name: Start triliovault object store service
+  service:
+    name: tvault-object-store.service
+    state: started
+  when: NFS == False
+  tags:
+    - update_cfg
+
+- name: Create {{VAULT_S3_SSL_CERT_DIR}} directory
+  file: name="{{VAULT_S3_SSL_CERT_DIR}}"  state=directory
+  when:
+      - VAULT_S3_SSL_CERT != ""
+      - S3|bool == True
+      - s3_type == Other_S3_Compatible
+  tags:
+    - update_cfg
+
+- name: check {{VAULT_S3_SSL_CERT}} is present or not
+  stat: path="{{VAULT_S3_SSL_CERT}}"
+  register: tvault_ssl_cert_output
+  when:
+      - VAULT_S3_SSL_CERT != ""
+      - S3|bool == True
+      - s3_type == Other_S3_Compatible
+  tags:
+    - update_cfg
+
+- name: Deploy user provided Triliovault s3 {{VAULT_S3_SSL_CERT}}
+  copy:
+    src: "{{VAULT_S3_SSL_CERT}}"
+    dest: "{{WLM_S3_SSL_CERT_PATH}}"
+  when:
+      - VAULT_S3_SSL_CERT != ""
```

Comment:
Use a variable which indicates if self singed certificates are needs to authenticate S3 backend. like VAULT_S3_SSL_ENABLED = True.
Then we need to use that variable here in ansible task instead of 'VAULT_S3_SSL_CERT'
Default value of this variable should be False.